doc/pacts/markdown/Pact Broker Client - Pact Broker.md

```diff
@@ -1089,6 +1093,14 @@ Pact Broker will respond with:
   },
   "body": {
     "description": "a webhook",
+    "request": {
+      "body": "{\"some\":\"body\"}",
```

Comment:
Thanks for updating these. The body should not be escaped however.

If you hop into the pact_broker project and run this, you'll see the diff:

```
be rake pact:verify:at[https://raw.githubusercontent.com/pact-foundation/pact_broker-client/b689da9375d01b88cb1372cd0fb70fe7558cf7e2/spec/pacts/pact_broker_client-pact_broker.json]
```

The events property needs to be up one level also.

You can generate the pact file and do the verify using the same command with the path to the local pact file.

Reply:
Thanks!